src/main/java/AddList.java

```diff
@@ -0,0 +1,71 @@
+import java.util.ArrayList;
+import java.util.List;
+import java.io.*;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+
+public class AddList {
+    List<Mission> ls;
+    int counter = 0;
```

Comment:
It's better to set these data members as private and do the initialization within constructor.